lms/templates/courseware/progress.html

```diff
@@ -56,10 +56,9 @@ <h2 class="hd hd-2 progress-certificates-title">
                     ${_("Course Progress for Student '{username}' ({email})").format(username=student.username, email=student.email)}
                 </h2>
 
-                %if certificate_data:
                 <div class="wrapper-msg wrapper-auto-cert">
                     <div id="errors-info" class="errors-info"></div>
-                    %if passed:
+                    %if certificate_data:
```

Comment:
Removing check for passed, because certificate_data will not return anything if passed is not true.

Reply:
I think the wrapper-msg has some padding around it that makes the progress page look wonky if there isn't any certificate data. I can move the endif to be at the right level though...